src/Raven.Server/Documents/PeriodicBackup/Azure/RavenAzureClient.cs

```diff
@@ -480,21 +483,174 @@ List<BlobProperties> GetResult()
                         .Select(x => new BlobProperties
                         {
                             Name = x
-                        })
-                        .ToList();
+                        });
                 }
-                else
+
+                return listBlobsResult
+                    .Descendants("Blob")
+                    .Select(x => new BlobProperties
+                    {
+                        Name = x.Element("Name")?.Value,
```

Comment:
This can return 'nulls' ? Are we expecting that?

Reply:
this is old code